saleor/csv/tests/export/products_data/test_handle_relations_data.py

```diff
@@ -102,6 +145,21 @@ def test_prepare_products_relations_data(product_with_image, collection_list):
         header = f"{assigned_attribute.attribute.slug} (product attribute)"
         expected_result[pk][header] = assigned_attribute.values.first().slug
 
+    for channel_listing in product_with_image.channel_listing.all():
+        if channel_listing.channel in [channel_PLN, channel_USD]:
+            channel_slug = channel_listing.channel.slug
+            for lookup, field in [
+                ("currency_code", "currency code"),
+                ("is_published", "published"),
+                ("publication_date", "publication date"),
+            ]:
+                header = f"{channel_slug} (channel {field})"
+                if lookup == "currency_code":
+                    value = getattr(channel_listing.channel, lookup)
```

Comment:
We have `currency` inside `channel_listing` maybe we can use them instead of getting a channel for every `channel_listing` ?